Example/CustomViewController.swift

```diff
@@ -8,7 +8,7 @@ class CustomViewController: UIViewController {
     
     var destinationAnnotation: PointAnnotation! {
         didSet {
-            navigationMapView.mapView.annotations.addAnnotation(destinationAnnotation)
+//            navigationMapView.mapView.annotations.addAnnotation(destinationAnnotation)
```

Comment:
Looks like we’ll need to use the AnnotationOrchestrator to create a PointAnnotationManager before calling `syncAnnotations(_:)` on it. Interestingly, NavigationMapView will have to (and will have the opportunity to) keep track of the annotations directly.